src/foam/dao/Relationship.js

```diff
@@ -685,10 +685,11 @@ foam.CLASS({
         name: this.propertyName,
         of: 'foam.dao.ManyToManyRelationship',
         flags: ['swift', 'js'],
-        getter: function() {
+        transient: true,
+        factory: function() {
           return this[methodName](this.__context__);
```

Comment:
If we switch this to a factory we also need to make sure we clear the property when id properties are changed, or make it an expression on id.

Reply:
Good call. Done.